schema/data/fixtures/form-errors-prevent-submit-setup.sql

```diff
@@ -0,0 +1,6 @@
+begin;
+alter table ggircs_portal.ciip_user_organisation
+  disable trigger _set_user_id;
+insert into ggircs_portal.ciip_user_organisation(user_id, organisation_id, status) values (6, 7, 'approved');
+update ggircs_portal.application_revision set legal_disclaimer_accepted = false where application_id=2 and version_number=1;
```

Comment:
Can you add a comment describing what this fixture does?

Reply:
Also shouldn't `legal_disclaimer_accepted` be set to `true`?

Reply:
yes, `legal_disclaimer_accepted` should be set to true.
comments break the tests as cy.exec() reads the comments

Reply:
comment problem fixed with a 'here-string' in the cy.exec() call